api/src/main/java/marquez/api/OpenSearchResource.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2018-2024 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.api;
```

Comment:
I would move the `pkg` to `marquez.api.v2beta.*` until we promote the API to `v2`.